manifests/w/WestWind/MarkdownMonster/3.3.15.0/WestWind.MarkdownMonster.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: WestWind.MarkdownMonster
+PackageVersion: 3.3.15.0
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Commands:
+- markdownmonster
+- mm
+- mmcli
+FileExtensions:
+- markdown
+- md
+- mdcrypt
+- mdproj
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
+    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
```

Comment:
```suggestion
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Reply:
I think this is a good suggestion to ensure all the features work as expected.